openwisp_controller/config/admin.py

```diff
@@ -52,6 +52,9 @@
 else:  # pragma: nocover
     from django.contrib.admin import ModelAdmin
 
+Device._meta.verbose_name = app_settings.DEVICE_VERBOSE_NAME
+Device._meta.verbose_name_plural = Device._meta.verbose_name + 's'
```

Comment:
this should go here: https://github.com/openwisp/openwisp-controller/blob/master/openwisp_controller/config/base/device.py#L79-L85

I see that you had an interaction with @NoumbissiValere and @pandafy about it, but this makes the code harder for a developer working with OpenWISP, they will have a hard time finding these two lines and then they'll be pretty surprised to find these here.
The right approach is to define this at model level and edit the migration so the migration also uses the setting instead of the raw value.

It would also be better to use a tuple in which the first element is the singular form and the second one the plural form, just to be sure to support corner cases (irregular forms, other languages).